metadata-ingestion/src/datahub/ingestion/source/sigma/data_classes.py

```diff
@@ -24,15 +24,19 @@ class Workspace(BaseModel):
 
 class SigmaDataset(BaseModel):
     datasetId: str
+    workspaceId: str
     name: str
     description: str
     createdBy: str
     createdAt: datetime
     updatedAt: datetime
     url: str
+    path: str
```

Comment:
would be better to keep this as a `List[str]` - that way if someone has a `/` in their folder name, we still handle it correctly

Reply:
We are getting path attribute details from API in string format. If we later convert it to list of string with any one folder containing `/` in name, still that folder name will get split.